tasks/pkg-windows-opts.yml

```diff
@@ -85,4 +85,11 @@
 
 - name: Set Test
   set_fact:
-    win_install_args: "{{ win_install_args }}"
\ No newline at end of file
+    win_install_args: "{{ win_install_args }}"
+
+# Add the installation arguments if Windows NPM is to be installed
+- name: Set Windows NPM flag
+  set_fact:
+    win_install_args: "{{ win_install_args }} NPM=true"
```

Comment:
```suggestion
    win_install_args: "{{ win_install_args }} ADDLOCAL=MainApplication,NPM"
```

Reply:
since ansible doesn't do reinstalls, this change isn't necessary.

Reply:
Might be still worth for consistency with the other config mgmt tools?